modules/Bio/EnsEMBL/IO/Parser/Bed.pm

```diff
@@ -102,8 +102,8 @@ sub add_format {
 =cut
 
 sub get_raw_chrom {
-  my $self = shift;
-  my $index = $self->{'column_map'}{'chrom'};
+  my ($self, $index) = @_;
+  $index ||= $self->{'column_map'}{'chrom'};
```

Comment:
This means if $index = 0 is provided, it will behave as if no $index has been provided.
Is this the required behaviour?

Reply:
if a provided $index is to be used always and the default value used only when $index is undef, then // should be used instead of ||.

Reply:
Changes made and tests added